tests/console/btrfs_qgroups.pm

```diff
@@ -65,6 +65,14 @@ sub run() {
     }
     assert_script_run "cp nofile c/nofile";
 
+    # Check for bsc#993841
+    assert_script_run "btrfs subvolume create e";
+    assert_script_run "btrfs qgroup limit 50m e .";
+    assert_script_run "! for c in {1..40}; do dd if=/dev/zero bs=1M count=40 of=e/file; done";
+    script_run "sync";
+    assert_script_run "rm e/file";
+    assert_script_run "touch e/newfile";
```

Comment:
If this is a check for bsc#993841 I would recommend to use `script_run`, check the return value and die with an explicit reference to the bug. `record_soft_failure` might be an option but I guess there is not much point in continuing on the borked filesystem in the next test module so it's better to die and revert to the last snapshot, right?